frontend/vite.config.mjs

```diff
@@ -0,0 +1,44 @@
+import { defineConfig } from 'vite'
+import inject from "@rollup/plugin-inject";
+import vue from '@vitejs/plugin-vue'
+
+export default defineConfig({
+   define: {
+      // TODO: https://github.com/vitejs/vite/issues/1973
+      'process.env': {}
+   },
+   transpileDependencies: ["bootstrap-material-design"],
+   build: {
+      commonjsOptions: { transformMixedEsModules: true },
+   },
+   plugins: [
+      vue({
+         template: {
+            compilerOptions: {
+               // PREvant's UI is ported to Vue3 but the vue-virtual-scroller isn't.
```

Comment:
https://github.com/Akryum/vue-virtual-scroller/tree/next

I thought we use [v2.0.0-beta.8](https://github.com/Akryum/vue-virtual-scroller/releases/tag/v2.0.0-beta.8)
And this is already ported to vue 3.

Reply:
Thanks. I think I was confused by other stuff going on. I could get rid of some other workarounds in this file too.